### PR TITLE
Update windows-store-for-business.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about Microsoft Store for Business being retired in 2023

Reference- https://techcommunity.microsoft.com/t5/windows-it-pro-blog/evolving-the-microsoft-store-for-business-and-education/ba-p/2569423

### CODE REVIEW

**Comment by PRMerger18** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2690

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/apps/windows-store-for-business.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ The [Microsoft Store for Business](https://www.microsoft.com/business-store) giv
 
 ## Before you start
 
+> [!IMPORTANT]
+> Microsoft Store for Business will be retired in the first quarter of 2023. However, admins can still leverage the connection to Store for Business and Education, from their UEM solution to deploy apps to managed Windows 11 devices, until they are retired in 2023.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/windows-store-for-business.md:48]:
```suggestion
> Microsoft Store for Business will be retired in the first quarter of 2023. However, admins can still leverage the connection to Store for Business and Education from their UEM solution to deploy apps to managed Windows 11 devices until they are retired in 2023. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger-2** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and SignOff this PR. Thanks

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**